src/react/createConnectedStoreAs.tsx

```diff
@@ -65,37 +66,36 @@ export function createConnectedStoreAs<
         fx(stores as any) // TODO
       }
 
+      this.storeDefinitions = stores as any
+      this.subscriptions = this._createSubscriptions()
       this.state = {
-        storeDefinitions: stores as any, // TODO
         // TODO
-        storeSnapshots: mapValues(stores, _ => _.getCurrentSnapshot()) as any,
-        subscriptions: mapValues(stores, (_, k) =>
-          _.onAll().subscribe(() =>
-            this.setState(state => ({
-              storeSnapshots: Object.assign({}, state.storeSnapshots, {
-                [k]: _.getCurrentSnapshot()
-              })
-            }))
-          )
+        storeSnapshots: mapValues(stores, _ => _.getCurrentSnapshot()) as any
+      }
+    }
+
+    _createSubscriptions = () =>
+      mapValues(this.storeDefinitions, (_, k) =>
+        _.onAll().subscribe(() =>
+          this.setState(state => ({
+            storeSnapshots: Object.assign({}, state.storeSnapshots, {
+              [k]: _.getCurrentSnapshot()
+            })
+          }))
         )
+      ) as any
```

Comment:
Nit: Do you need this `as any`?

Reply:
I can double check, I feel like I had basically just moved these lines but could see if I can fix at least some of them.